### PR TITLE
documentation changes

### PR DESCRIPTION
**Pull request by rvguha**
Changes to datasets doc, about doc

### CODE REVIEW

**Comment by rvguha** [comment]
@googlebot I signed it!

File: server/templates/static/about.html [hunk 1]
```diff
@@ -24,39 +24,77 @@
...
+   Majumdar</a>, Director, Precourt Institute for Energy.
+   </ul>
+    
+    
 
     <section>
       <h3>See also</h3>
       <ul>
-        <li><a href="{{ url_for('static.datasets') }}">Ingested datasets</a>
+        <li><a href="{{ url_for('static.datasets') }}">Datasets</a>
```
Comment (beets) [inline server/templates/static/about.html:95]: Should we call this Data Sources too? I can also update the URL with a redirect as a follow up.

File: server/templates/static/datasets.html [hunk 2]
```diff
@@ -47,369 +49,192 @@ <h1>Datasets in the Data Commons Graph</h1>
...
-            Service</a>.
-        </p>
+<h3><a href="https://census.gov">Census Bureau</a></h3>
+<ul>
+<li><a href="https://www.census.gov/programs-surveys/acs">American
+            Community            Survey  </a>
+        covers a broad range of topics about social, economic, demographic,  and
+          housing characteristics of the U.S. population. The ACS
+5-year (and 1-year) estimates are updated every year, based on the last
+          5 years (q year) of collected data. Data Commons includes thousands of variables across the full range of ACS topics at the
```
Comment (beets) [inline server/templates/static/datasets.html:128]: what is q year

File: server/templates/static/datasets.html [hunk 3]
```diff
@@ -47,369 +49,192 @@ <h1>Datasets in the Data Commons Graph</h1>
...
-          counties, metropolitan statistical areas, and zip codes.
-        </p>
-        <p> Data made available under <a href="https://www.census.gov/data/developers/about/terms-of-service.html">US
-            Census
-            Terms
-            of Service</a>.
-        </p>
+        <li><a href="https://www.census.gov/programs-surveys/cbp/data/datasets.html">County Business
+            Patterns</a></li>
+    Pper-industry number of establishments, employment, payroll and
```
Comment (beets) [inline server/templates/static/datasets.html:147]: Nit: Pper-industry

File: server/templates/static/datasets.html [hunk 4]
```diff
@@ -47,369 +49,192 @@ <h1>Datasets in the Data Commons Graph</h1>
...
         </p>
-        <p>Please see the <a href="https://datacommons.org/disclaimers">disclaimers</a> page about the scope of the
-          data,
-          as
-          well as the <a href="https://www.justice.gov/legalpolicies">US Department of Justice Legal Policies and
-            Disclaimers
-            Terms of Use</a>.
-        </p>
+
+	    <!---
```
Comment (beets) [inline server/templates/static/datasets.html:237]: are we leaving this out on purpose?

File: server/templates/static/datasets.html [hunk 5]
```diff
@@ -422,210 +247,166 @@ <h3><a href="https://www.eia.gov/electricity/data/eia860/">Energy Information Ad
...
         </p>
         <p> Data made available under the <a href="https://www.eia.gov/about/copyrights_reuse.php">public domain</a>.
-        </p>
+        </p> -->
 
-        <h3><a href="https://www.fbi.gov/services/cjis/ucr">Federal Bureau of Investigation Uniform Crime Reporting
+	  <h3><a href="https://fbi.gov">Federal Bureau of
+	  Investigation</a></h3>
+	    <ul>
+        <li><a href="https://www.fbi.gov/services/cjis/ucr">FBIUniform Crime Reporting
```
Comment (beets) [inline server/templates/static/datasets.html:255]: add a space after FBI?

File: server/templates/static/datasets.html [hunk 6]
```diff
@@ -422,210 +247,166 @@ <h3><a href="https://www.eia.gov/electricity/data/eia860/">Energy Information Ad
...
-            and
-            Credits
-            Terms of Service</a>.
+            and Credits Terms of Service</a>.
         </p>
 
       </article> <!-- end of id="federal-govt-data" -->
       <article id="state-govt-data">
 
+	    <!--
```
Comment (beets) [inline server/templates/static/datasets.html:380]: are we leaving this out on purpose as well?

**Review by beets — COMMENTED** [review]
thanks guha! a few small questions on some of the text updates

File: server/templates/browser/kg_main.html [hunk 7]
```diff
@@ -27,11 +27,11 @@
 
 {% block content %}
   <div class="container">
-    <h1 class="mb-4">Data Commons Graph Browser</h1>
+ <!--   <h1 class="mb-4">Data Commons Graph Browser</h1> -->
```
[the remark this answers is not in the document]
Comment (rvguha) [reply server/templates/browser/kg_main.html:30]: yes

File: server/templates/static/about.html [hunk 8]
```diff
@@ -24,39 +24,77 @@
...
+    Data Commons has benefited greatly from many collaborations. In
+    addition to help from US Department of Commerce (notably the
+    Census Bureau), we have received help from our many academic
+    collaborations, including, UC San Francisco, Stanford University,
+UC Berkeley and Harvard. 
+</p>
+<p>
+We are looking for more collaborators, both for adding new data to
+Data Commons and for building new and interesting applications of Data
+      Commons. Contact us if you are interested in working with us.
```
[the remark this answers is not in the document]
Comment (rvguha) [reply server/templates/static/about.html:73]: we should create a new email ...

File: server/templates/static/about.html (hunk 1 above)
Comment (rvguha) [reply server/templates/static/about.html:95]: yes, made the change

File: server/templates/static/datasets.html (hunk 2 above)
Comment (rvguha) [reply server/templates/static/datasets.html:135]: fixed

File: server/templates/static/datasets.html (hunk 3 above)
Comment (rvguha) [reply server/templates/static/datasets.html:154]: fixed

File: server/templates/static/datasets.html (hunk 4 above)
Comment (rvguha) [reply server/templates/static/datasets.html:244]: yes

File: server/templates/static/datasets.html (hunk 5 above)
Comment (rvguha) [reply server/templates/static/datasets.html:262]: fixed

File: server/templates/static/datasets.html (hunk 6 above)
Comment (rvguha) [reply server/templates/static/datasets.html:387]: yes

**Review by beets — APPROVED** [review]

**Pull request merged**